2022q1/rtw88.adoc

```diff
@@ -0,0 +1,24 @@
+=== Realtek Wireless driver support
+
+Links: +
+link:https://wiki.freebsd.org/WiFi/Rtw88[rtw88 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw88[https://wiki.freebsd.org/WiFi/Rtw88]
+link:https://wiki.freebsd.org/WiFi/Rtw89[rtw89 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw89[https://wiki.freebsd.org/WiFi/Rtw89]
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.ORG>
+
+While the Intel Wireless driver update project is the main driver behind the work to bring support for newer chipsets and eventually newer IEEE 802.11 standards support, there is also an ongoing effort to support more drivers.
+The next two drivers in the (already longer) queue are Realtek's rtw88 and rtw89.
+
+While the initial driver porting efforts for rtw88 and rtw89 happened on personal time, the LinuxKPI integration has to be done more and more along the Intel wireless driver work and so thanks are also due to the FreeBSD Foundation.
+
+The rtw88 driver has started to work on machines with less than 4GB of main memory and should (despite that problem) be in the FreeBSD git repository by the time this report is published.
+While our version of the driver is aware of these limitations, the problem is currently assumed to be outside the driver in the interactions with LinuxKPI and busdma.
+
+The rtw89 driver has happily started to send packets and has troubles receiving at this point.
+More investigations will happen as soon as rtw88 is sorted out and it is expected that rtw89 will then also timely follow into FreeBSD's git repository.
```

Comment:
I think you want the collective singular ("investigation") here unless you already know there are unrelated things that will need separate investigations, one for each.